Automattic/jetpack
# PHPCS: remove errors in Portfolios CPT

See #21963

#### Changes proposed in this Pull Request:

* Fix PHPCS errors in Portfolios.
* This also refactors the way the shortcode attributes are handled a bit, so we avoid having to handle too many PHPCS situations based on that. I'll let you judge whether this is overkill or not.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Go to Jetpack > Settings > Writing and enable Portfolios
* Go to the new Portfolios menu and create a new porfolio items.
* Go to Posts > Add New, and add a new Portfolio shortcode using the shortcode block: `[portfolio columns=5 display_types=false orderby=rand display_content=false]`
* Test [different variations](https://wordpress.com/support/portfolios/portfolio-shortcode/#customization-options), see how it goes.
